src/components/application_manager/include/application_manager/resumption/resumption_data_processor_impl.h

```diff
@@ -134,6 +134,7 @@ class ResumptionDataProcessorImpl
   bool IsResumptionFinished(const uint32_t app_id,
                             const ResumptionRequest& found_request);
 
+  bool IsResumptionFinished(const uint32_t app_id);
```

Comment:
@ychernysheva function can be const

Reply:
@AKalinich-Luxoft I think that we cannot do that because `operator[]` is used inside of this function, and also lock (which may me `mutable`, but I think that all this changes are redundant in this case).